test/time-picker/time-picker-spec.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import TextField from 'text-field';
+import TimePicker from 'time-picker/time-picker';
+import injectTheme from '../fixtures/inject-theme';
+import DateTime from 'utils/date-time';
+import TestUtils from 'react-addons-test-utils';
+
+describe('TimePicker', () => {
+  let ThemeTimePicker;
+
+  beforeEach(() => {
+    ThemeTimePicker = injectTheme(TimePicker);
```

Comment:
Our components work out of the box without the need to inject a theme. Can you remove it and simply use `TimePicker` instead of `ThemeTimePicker`?